### PR TITLE
ABOUT_THESE_NOTEBOOKS.md as a suggested order of approach

### PR DESCRIPTION
**Pull request by brianmapes**
That directory is a little blank -- just alphabetical. Could use a bit of a front face. Not sure if this is the right way, but a suggestion.

### CODE REVIEW

**Comment by dopplershift** [comment]
This is a nice outline to have, thanks!

@jrleeman if you have a place where you think this would fit in the rendered docs, I'm all ears. Regardless, this is a nice thing to have alongside the notebooks.

File: environment.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@
    - pip
    - ffmpeg
    - pytest
+   - ipywidgets
    - pip:
      - python-awips
      - mpldatacursor
+     - drilsdown
```
Comment (dopplershift) [inline environment.yml]: Brian, did you mean to push this? Because these don't look like changes we should take. If it was a mistake, we can handle.

**Comment by brianmapes** [comment]
Oh, no I didn’t mean to push that to your world.

I guess pull requests include too many things.




On Feb 5, 2018, at 4:16 PM, Ryan May <notifications@github.com<mailto:notifications@github.com>> wrote:


@dopplershift commented on this pull request.

________________________________

In environment.yml<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUnidata%2Funidata-python-workshop%2Fpull%2F267%23discussion_r166114103&data=02%7C01%7Cbmapes%40rsmas.miami.edu%7C84288caddaec4f09484808d56cddac32%7C6f24b93bfe274c65a17999f3ac4ce9c3%7C0%7C0%7C636534621690514676&sdata=jgMF6atMKTz2EEUAO5vtVB8S0KyZB9Hqm2UHcYXLq%2BI%3D&reserved=0>:

>     - pip:
      - python-awips
      - mpldatacursor
+     - drilsdown


Brian, did you mean to push this? Because these don't look like changes we should take. If it was a mistake, we can handle.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUnidata%2Funidata-python-workshop%2Fpull%2F267%23pullrequestreview-94144974&data=02%7C01%7Cbmapes%40rsmas.miami.edu%7C84288caddaec4f09484808d56cddac32%7C6f24b93bfe274c65a17999f3ac4ce9c3%7C0%7C0%7C636534621690514676&sdata=e2US8%2FHBMaQ7j2yVcUCvuu0tqVy%2FJYZurMYcTBoIfsI%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAB_VQs4EEzW2FQ-uu8mdUWz52NSOoKeOks5tR2-WgaJpZM4Ruhkp&data=02%7C01%7Cbmapes%40rsmas.miami.edu%7C84288caddaec4f09484808d56cddac32%7C6f24b93bfe274c65a17999f3ac4ce9c3%7C0%7C0%7C636534621690514676&sdata=q%2BDP9iB5b8hpK90dK9HpCHNY7F99PH1qU34M8PllAS0%3D&reserved=0>.

*********************************************
Brian Mapes, Professor
Department of Atmospheric Sciences
Meteorology and Physical Oceanography Program
RSMAS, University of Miami
4600 Rickenbacker Causeway
Miami, FL 33149-1031

phone: (305) 421-4275
fax: (305) 421-4696
email: mapes@miami.edu<mailto:mapes@miami.edu>
Web: http://www.rsmas.miami.edu/users/bmapes/
**********************************************

**Comment by brianmapes** [comment]
In fact, I think I I only added that to my fork today.

Do standing pull requests get bigger with further edits of a fork?


Brian




On Feb 5, 2018, at 7:05 PM, Mapes, Brian Earle <bmapes@rsmas.miami.edu<mailto:bmapes@rsmas.miami.edu>> wrote:

Oh, no I didn’t mean to push that to your world.

I guess pull requests include too many things.




On Feb 5, 2018, at 4:16 PM, Ryan May <notifications@github.com<mailto:notifications@github.com>> wrote:


@dopplershift commented on this pull request.

________________________________

In environment.yml<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUnidata%2Funidata-python-workshop%2Fpull%2F267%23discussion_r166114103&data=02%7C01%7Cbmapes%40rsmas.miami.edu%7C84288caddaec4f09484808d56cddac32%7C6f24b93bfe274c65a17999f3ac4ce9c3%7C0%7C0%7C636534621690514676&sdata=jgMF6atMKTz2EEUAO5vtVB8S0KyZB9Hqm2UHcYXLq%2BI%3D&reserved=0>:

>     - pip:
      - python-awips
      - mpldatacursor
+     - drilsdown


Brian, did you mean to push this? Because these don't look like changes we should take. If it was a mistake, we can handle.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUnidata%2Funidata-python-workshop%2Fpull%2F267%23pullrequestreview-94144974&data=02%7C01%7Cbmapes%40rsmas.miami.edu%7C84288caddaec4f09484808d56cddac32%7C6f24b93bfe274c65a17999f3ac4ce9c3%7C0%7C0%7C636534621690514676&sdata=e2US8%2FHBMaQ7j2yVcUCvuu0tqVy%2FJYZurMYcTBoIfsI%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAB_VQs4EEzW2FQ-uu8mdUWz52NSOoKeOks5tR2-WgaJpZM4Ruhkp&data=02%7C01%7Cbmapes%40rsmas.miami.edu%7C84288caddaec4f09484808d56cddac32%7C6f24b93bfe274c65a17999f3ac4ce9c3%7C0%7C0%7C636534621690514676&sdata=q%2BDP9iB5b8hpK90dK9HpCHNY7F99PH1qU34M8PllAS0%3D&reserved=0>.

*********************************************
Brian Mapes, Professor
Department of Atmospheric Sciences
Meteorology and Physical Oceanography Program
RSMAS, University of Miami
4600 Rickenbacker Causeway
Miami, FL 33149-1031

phone: (305) 421-4275
fax: (305) 421-4696
email: mapes@miami.edu<mailto:mapes@miami.edu>
Web: http://www.rsmas.miami.edu/users/bmapes/
**********************************************




*********************************************
Brian Mapes, Professor
Department of Atmospheric Sciences
Meteorology and Physical Oceanography Program
RSMAS, University of Miami
4600 Rickenbacker Causeway
Miami, FL 33149-1031

phone: (305) 421-4275
fax: (305) 421-4696
email: mapes@miami.edu<mailto:mapes@miami.edu>
Web: http://www.rsmas.miami.edu/users/bmapes/
**********************************************

**Comment by dopplershift** [comment]
When you create a pull request, GitHub essentially creates a link between the pull request and the branch you are asking to be "pulled in"--in this case the master branch on your fork of the python workshop materials. Every time you push to that branch, this pull request is updated.

That's a great thing for when we're collaboratively going back and forth on changes, since it means you can easily update a pull request. It's less great in this case. At any rate, that link is severed once the PR is merged, which means it's really our fault for not merging in your contribution in a timely fashion.

**Comment by jrleeman** [comment]
@brianmapes - is it okay if I blast away that environment.yml change? It will go away on your master branch as I would be pushing there.

**Comment by brianmapes** [comment]
Yes that’s fine.


On Mar 19, 2018, at 12:53 PM, John Leeman <notifications@github.com<mailto:notifications@github.com>> wrote:


@brianmapes<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbrianmapes&data=02%7C01%7Cbmapes%40rsmas.miami.edu%7C149e342d59104553e7f708d58dba0468%7C2a144b72f23942d48c0e6f0f17c48e33%7C0%7C0%7C636570752497113675&sdata=Jdf70HI%2B%2ByxWd%2FsVtraMaFfSSV1CjY9JIn8G5CLvtLM%3D&reserved=0> - is it okay if I blast away that environment.yml change? It will go away on your master branch as I would be pushing there.

**Pull request closed without merging**